tables/isolate_mutations.csv

```diff
@@ -12281,3 +12281,51 @@ SARS-CoV-2/human/DNK/7022055836/2022,N,33,del,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,203,K,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,204,R,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,413,R,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,19,R,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,142,D,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,156,G,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,157,del,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,158,del,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,417,N,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,452,R,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,478,K,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,614,G,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,681,R,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,950,N,NULL,NULL
+BA.1 Spike:+214ins,S,67,V,NULL,NULL
+BA.1 Spike:+214ins,S,69,del,NULL,NULL
+BA.1 Spike:+214ins,S,70,del,NULL,NULL
+BA.1 Spike:+214ins,S,95,I,NULL,NULL
+BA.1 Spike:+214ins,S,142,D,NULL,NULL
+BA.1 Spike:+214ins,S,143,del,NULL,NULL
+BA.1 Spike:+214ins,S,144,del,NULL,NULL
+BA.1 Spike:+214ins,S,145,del,NULL,NULL
+BA.1 Spike:+214ins,S,211,del,NULL,NULL
+BA.1 Spike:+214ins,S,212,ins,NULL,NULL

```

Comment:
Having 212ins and 214ins so close is almost definitely an alignment issue of the paper. It should be changed to 211I + 212del + 214ins no matter what the paper suggested. If this is the case, we don't need this isolate.

Reply:
This makes sense, I'll make the change